Sources/xcbeautify/Xcbeautify.swift

```diff
@@ -56,6 +59,10 @@ struct Xcbeautify: ParsableCommand {
             print("Took \(diff) seconds")
         }
         #endif
+        
+        if !disableVersionLogging {
+            print("Version: \(version)")
```

Comment:
Hi @ikelax. Thanks so much for making a contribution!

Can you include `xcbeautify` in this print statement? It will give context to users what "Version" we're referencing. I think it might also help to make this more prominent.

Consider something like...

```swift
print(
  """
  
  ----- xcbeautify -----
  Version: \(version)
  ----------------------
  
  """
)
```

Otherwise, this looks good to me! No need for an env var.

Reply:
Thank you for the review!

I included it. It looks great :)

Reply:
@ikelax can you please confirm --disable-version-logging is working?

Reply:
@ikelax can you please confirm --disable-version-logging is working?
$ xcbeautify
----- xcbeautify -----
Version: 2.4.0
----------------------

$ xcbeautify --disable-version-logging
Error: Unknown option '--disable-version-logging'
Usage: xcbeautify [--quiet] [--quieter] [--preserve-unbeautified] [--is-ci] [--disable-colored-output] [--disable-logging] [--renderer <renderer>] [--report <report> ...] [--report-path <report-path>] [--junit-report-filename <junit-report-filename>]
  See 'xcbeautify --help' for more information.